src/stores/VoiceRecordingStore.ts

```diff
@@ -46,46 +46,46 @@ export class VoiceRecordingStore extends AsyncStoreWithClient<IState> {
 
     /**
      * Gets the active recording instance, if any.
-     * @param {string} roomId The room ID to get the recording in.
+     * @param {string} voiceRecordingId The room ID (with optionnaly the thread ID if in one) to get the recording in.
```

Comment:
You’ve got a typo here
```suggestion
     * @param {string} voiceRecordingId The room ID (with optionally the thread ID if in one) to get the recording in.
```